### DISCUSSION TITLE
Will you pull code from upstream when Sway 1.1 releases?

### DISCUSSION THREAD

**Discussion by trustytrojan** [WillPower3309/swayfx#354] [Q&A] [answered]
Wondering if this will happen because sway 1.1 proposes many good feature additions, an important one being the screen tearing protocol

**Comment by RicArch97** [comment, marked as answer]
Yes, the idea is to always rebase on the latest version of Sway. SwayFX is really meant to be Sway + effects, so it tries to stay close to Sway.

Benefits of being a fork, can always pull in cool new features for free :)
